docs/satellite-clone.md

```diff
@@ -91,3 +91,24 @@ On the target server:
       # ansible-playbook satellite-clone-playbook.yml
     ```
   The playbook run may may take a while to complete.
+
+## Cloning a Satellite With Remote Databases
+
+Note: The backup used for these procedures must have the correct options to manage remote databases in `/etc/foreman-installer/scenarios.d/satellite-answers.yaml`. Please check that the answers file has the correct database management values filled in the configuration backup file. These will be values such as `db-manage` in the `foreman` section and `pulp-manage-db` in the `katello` section. They are currently (at the time of this writing) [missing from the initial setup documentation](https://bugzilla.redhat.com/show_bug.cgi?id=1887846)
+
+1. Set up the remote databases according to [Satellite documentation](https://access.redhat.com/documentation/en-us/red_hat_satellite/) in the same way that they were set up on your original Satellite.
+2. Set up a base RHEL7 server that will become the cloned satellite and make sure the target clone server has network access to the remote databases.
+3. The remote databases for the clone will also need to be reached on the same hostname as ones used for the original Satellite. You can use `/etc/hosts` on the target server to associate the original remote database hostnames with the new clone database IP addresses.
+4. Install satellite-clone on target server according to [instructions](#instructions), but don't run the playbook yet.
+5. In `satellite-clone-vars.yml` find and uncomment the line: `installer_additional_options: ""`
```

Comment:
Don't the installer opts come from last_scenario.yaml? IOW why would the user need to set these? If they must be set, I think it'd be nice to create the string during the playbook run, and let the user specify key-value pairs - just thinking about the UX here.

Reply:
Good question, I only tested out running things on a single server (no remote dbs) with the current sat-clone, but I didn't try it with remote db's set up properly. We do restore the config before the first install. 

I'll test it out now that we know how to set up the db's properly!